### ISSUE TITLE
About Folder Dataset

### ISSUE THREAD

**Issue by myukimm** [torcheeg/torcheeg#128]
I understand that when using a 'Folder Dataset', it automatically segments into 10 parts. However, looking at the GitHub code, it seems that I can modify this by setting the read_fn, but I am having difficulty finding the exact method. Could you provide some reference examples?

Here, my code .

def default_read_fn2(file_path, **kwargs):
    # Load EEG file
    raw = mne.io.read_raw(file_path)
    # Convert raw to epochs
    epochs = mne.make_fixed_length_epochs(raw, duration=10)
    # Return EEG data
    return epochs

label_map = {'label01': 0, 'label02': 1}
train_dataset = FolderDataset(
    root_path='/content/gdrive/MyDrive/test/',
    structure='subject_in_label',
    read_fn=default_read_fn2,
    online_transform=transforms.ToTensor(),
    label_transform=transforms.Compose([
        transforms.Select('label'),
        transforms.Lambda(lambda x: label_map[x])
    ]),
    num_worker=4
)
